css/css-text/text-transform/text-transform-copy-paste-001-manual.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>CSS Text 3 test: effects of text-transform on plain text copy&paste</title>
+<link rel=author title="Florian Rivoal" href="https://florian.rivoal.net">
+<link rel=help href="https://drafts.csswg.org/css-text-3/#text-transform-property">
+<meta name=flags content="interact">
+<meta name=assert content="text-transform must not affect the content of a plain text copy and paste operation.">
+<!--
+I wish this test could be written automatedly, but I don't think it can,
```

Comment:
If you could rewrite this as testharness.js, then you can use testdriver.js and `test_driver.bless()` to automate this. And I think you can, since you just need to read back the value?

Note that @gsnedders will also be working on making testdriver.js work with reftests in https://github.com/web-platform-tests/wpt/issues/13183.

Reply:
Well, `document.execCommand(‘copy’)` would benefit form `test_driver.bless()`, but also it's not quite clear whether it is standard or not. It is documented in https://w3c.github.io/editing/execCommand.html#the-copy-command-0, but that spec has a big "no done & not making progress" warning, and MDN calls it deprecated.

Also, even if you get past that, I then need to check the content of the clipboard, and I don't think here's a workable cross browser way to do that:
* `document.execCommand(‘paste’)` isn't supported in regular web pages by anyone but IE
* The alternative, the Clipboard API, is not supported across the board, and Firefox only supports reading the clipboard in browser extensions.

This may change eventually, but as far as I can tell, a manual test is still all we can do.